compiler/rustc_target/src/spec/targets/aarch64_apple_ios_macabi.rs

```diff
@@ -6,7 +6,7 @@ pub(crate) fn target() -> Target {
     Target {
         llvm_target,
         metadata: crate::spec::TargetMetadata {
-            description: Some("Apple Catalyst on ARM64".into()),
+            description: Some("ARM64 Apple Catalyst".into()),
```

Comment:
Btw, I felt "Apple Catalyst" is a bit odd. AFAIK Apple docs call it "Mac Catalyst", not just "Catalyst".
https://developer.apple.com/documentation/uikit/mac_catalyst/

So maybe "Apple Mac Catalyst" is better here.
(x86_64_apple_ios_macabi.rs also has the same issue)

Reply:
Agreed

Reply:
Works for me :)